### DISCUSSION TITLE
Where to ask project related doubts.

### DISCUSSION THREAD

**Discussion by XxThunderBlastxX** [community/community#49952] [GitHub Education]
I have shown interest in Houseware- Authorization and Authentication project. And in the README.md it is written to ask project related issues, I shall ask in Github Discussions. But there is no tab available for that repo as "Discussions". So Where or how shall I ask the doubts related to the given task ??

**Comment by samus-aran** [comment]
Hi @XxThunderBlastxX 👋 

I would ask in an issue if there are no discussions enabled for that repo. I'm sure the team will provide guidance if issues are the best place once you create one 👍

**Reply by XxThunderBlastxX** [reply]
Yeah I asked by opening an issue . But still no response for in that issue. It has been more than 24hrs

**Comment by datagrains** [comment]
In the repo, go into 'Issues' and click 'New issue'

![image](https://user-images.githubusercontent.com/30798870/225398453-ecc32f4f-3772-4a00-9ab5-c41cef5abcba.png)

If they are not responding and your issue is urgent, I would considering DMing them on GitHub or drop them a message if they provided an email address.  It's possible they have a huge backlog so aren't able to respond immediately to your question :)

**Reply by XxThunderBlastxX** [reply]
Yeah that could be the problem . Thank you for your suggestion :)
